server/src/domain/system-config/system-config.repository.ts

```diff
@@ -7,4 +8,5 @@ export interface ISystemConfigRepository {
   readFile(filename: string): Promise<Buffer>;
   saveAll(items: SystemConfigEntity[]): Promise<SystemConfigEntity[]>;
   deleteKeys(keys: string[]): Promise<void>;
+  getLatestAvailableVersion(): Promise<GithubRelease>;
```

Comment:
Getting the latest release information from github has nothing to do with system configuration.